app/assets/images/payment_icons/sofort.svg

```diff
@@ -1 +1 @@
1.062-.264-1.362-.672l.564-.57c.168.216.444.45.81.45.342 0 .576-.204.576-.528 0-.348-.276-.432-.714-.558-.804-.234-1.128-.654-1.128-1.194 0-.678.516-1.266 1.332-1.266.516 0 .99.24 1.2.708l-.678.426c-.126-.21-.306-.348-.546-.348-.282 0-.462.192-.462.438 0 .294.258.384.672.504.648.192 1.176.516 1.176 1.26 0 .804-.624 1.35-1.44 1.35zm3.298 0c-.864 0-1.566-.69-1.566-1.53 0-.84.702-1.53 1.566-1.53.864 0 1.566.69 1.566 1.53 0 .84-.702 1.53-1.566 1.53zm0-.768c.42 0 .75-.33.75-.762a.745.745 0 0 0-.75-.762c-.42 0-.75.33-.75.762s.33.762.75.762zm3.586-2.742c-.456-.018-.786.21-.786.714v.312c.18-.192.438-.288.762-.288v.78c-.45.012-.762.3-.762.702v1.212h-.816v-2.76c0-.78.486-1.44 1.482-1.44h.12v.768zm1.78 3.51c-.864 0-1.566-.69-1.566-1.53 0-.84.702-1.53 1.566-1.53.864 0 1.566.69 1.566 1.53 0 .84-.702 1.53-1.566 1.53zm0-.768c.42 0 .75-.33.75-.762a.745.745 0 0 0-.75-.762c-.42 0-.75.33-.75.762s.33.762.75.762zm2.8-1.884c.162-.264.426-.39.798-.378v.876c-.492 0-.774.27-.774.75v1.326h-.816v-2.904h.792v.33zm2.08 1.092c0 .504.33.732.786.714v.768h-.12c-.996 0-1.482-.66-1.482-1.44v-2.4h.816v.456c0 .396.318.69.762.702v.78c-.324 0-.582-.096-.762-.288v.708zm1.762 1.53a.548.548 0 0 1-.546-.546c0-.3.246-.546.546-.546.3 0 .546.246.546.546 0 .3-.246.546-.546.546zm-7.772-6.71c2.55-.004 3.628-1.459 3.824-3.789.056-.67-.056-1.079-.258-1.295a.486.486 0 0 0-.153-.114.23.23 0 0 0-.06-.018l-3.023-.706a1.276 1.276 0 0 0-.581-.02.57.57 0 0 0-.46.431c-.101.34-.094.6.121.807.056.02.101.057.132.104.15.1.362.186.648.264l1.993.445c.33.074.23.57-.104.507a1.76 1.76 0 0 0-.86.083c-.487.183-.789.601-.826 1.377a.259.259 0 0 1-.518-.026c.042-.855.376-1.421.915-1.722l-.724-.161a3.181 3.181 0 0 1-.635-.24c-.083.128-.192.24-.321.33l-.057.037c-.035.024-.071.045-.107.065.23.434.167.983-.191 1.355l-.05.054a1.17 1.17 0 0 1-1.65.022l-.388-.384c.318 1.553 1.702 2.593 3.32 2.593h.013zm-.09.516c-2.05-.037-3.773-1.547-3.828-3.69l-.471-.466c-.582-.584-.676-1.393-.208-1.873l.05-.053c.2-.207.464-.32.733-.334-.325-.641-.224-1.343.282-1.69l.072-.047c.55-.319 1.028-.196 1.437.21l-.47-3.865c-.158-.672.254-1.408.882-1.542a1.233 1.233 0 0 1 1.464.958l.801 3.42 1.015-3.476a1.285 1.285 0 0 1 1.565-.911c.617.183.989.811.854 1.436l-1.118 5.57.48.112c.14.02.322.103.494.287.313.334.464.886.396 1.692-.217 2.59-1.493 4.288-4.413 4.263l-.017-.001zm2.556-6.47L22.918 3.2a.711.711 0 0 0-.484-.832.767.767 0 0 0-.928.55l-1.292 4.427a.26.26 0 0 1-.433.108.258.258 0 0 1-.07-.124l-1.026-4.384a.713.713 0 0 0-.85-.56c-.334.07-.584.523-.483.926l.005.031.583 4.79.37.552c.16-.22.39-.358.664-.419a1.773 1.773 0 0 1 .824.024l2.006.468zm-3.726.51l-.572-.852a.258.258 0 0 1-.053-.079l-.262-.39c-.334-.486-.658-.654-1.04-.433l-.055.037c-.297.204-.338.698-.028 1.167l1.132 1.685a.651.651 0 0 0 .902.173l.056-.037a.64.64 0 0 0 .19-.205c-.282-.293-.35-.655-.27-1.067zm-.324 1.937a1.17 1.17 0 0 1-.983-.512l-.842-1.253a.587.587 0 0 0-.765.065l-.05.053c-.252.259-.196.75.2 1.147l1.44 1.427a.65.65 0 0 0 .915-.012l.05-.052a.65.65 0 0 0 .035-.863z"/></g></svg>
```

Comment:
`<path fill="#FFF"` sets to white now 😬 

![https://screenshot.click/02-19-20dog-gfm80.jpg](https://screenshot.click/02-19-20dog-gfm80.jpg)